UPGRADING.md

```diff
@@ -315,6 +315,31 @@ message MsgSetWithdrawAddress {
 }
 ```
 
+#### `{accepts,implements}_interface` proto annotations
+
+The SDK is normalizing the strings inside the Protobuf `accepts_interface` and `implements_interface` annotations. We require them to be fully-scoped names. They will soon be used by code generators like Pulsar and Telescope to match which messages can or cannot be packed inside `Any`s.
+
+Here are the following replacements that you need to perform on your proto files:
+
+```diff
+- "Content"
++ "cosmos.gov.v1beta1.Content"
+- "Authorization"
++ "cosmos.authz.v1beta1.Authorization"
+- "sdk.Msg"
++ "cosmos.base.v1beta1.Msg"
+- "AccountI"
++ "cosmos.auth.v1beta1.AccountI"
+- "ModuleAccountI"
++ "cosmos.auth.v1beta1.ModuleAccountI"
+- "FeeAllowanceI"
++ "cosmos.feegrant.v1beta1.FeeAllowanceI"
+```
+
+Please also check that in your own app's proto files that there are no single-word names for those two proto annotations. If so, then replace them with fully-qualified names, even though those names don't actually resolve to an actual protobuf entity.
```

Comment:
With @pyramation, we decided to always use fully-scoped names. However, these names don't map to anything in protobuf. Is that fine?

There's an optional proposal to map them to some human-readable description here: https://github.com/cosmos/cosmos-sdk/pull/14280/files#r1047005098

Reply:
IMO it's completely fine, but we can have some docs if necessary.

Reply:
docs that may need updating after this change:

- [ ] https://github.com/regen-network/cosmos-proto#implements_interface
- [ ] https://github.com/cosmos/cosmos-sdk/blob/main/docs/docs/core/05-encoding.md#interface-encoding-and-usage-of-any

Also, would be great if there is an ADR somewhere that may need updating? If we can describe that 1. these interfaces need to be fully-scoped, and also 2. won't be defined in any proto files, just so it's clear, that would be great.

I'd love to share the updated docs with a few teams and folks like @faddat and his team, so we can update protos everywhere.